core/src/main/scala/tech/beshu/ror/accesscontrol/logging/LoggingContext.scala

```diff
@@ -0,0 +1,6 @@
+package tech.beshu.ror.accesscontrol.logging
+
+import cats.Show
+import tech.beshu.ror.accesscontrol.domain.Header
+
+final class LoggingContext(implicit val showHeader:Show[Header])
```

Comment:
IMO we should store in LoggingContext data, not typeclasses. I mean, IMO it's better to defined is like this:
`final class LoggingContext(obfuscatedHeaders: Set[Header.Name])`

Then you can use this class not only to stringify headers, but also to filter them, etc.